src/lib/components/collections/req-res-section/sub-components/response-body-section/ResponseBody.svelte

```diff
@@ -155,7 +155,7 @@
       <div class="d-flex gap-2">
         <button
           class="statuscode position-relative cursor-pointer ps-1 pe-1 border-0 rounded d-flex align-items-center justify-content-center text-backgroundColor gap-1 {statusCode ===
-          '200 OK'
+            '200 OK' || statusCode === '201 Created'

```

Comment:
Lets put these in constant file. Maybe make a httpStatusCode.enum.ts

Reply:
Resolved